src/main.rs

```diff
@@ -228,6 +231,17 @@ struct DependencySpec {
 
 fn main() -> std::result::Result<(), Box<dyn std::error::Error + 'static>> {
     let mut flags = Flags::from_args();
+
+    let version = option_env!("CARGO_PKG_VERSION").unwrap_or_else(|| "0.0.0").to_string();
+
+    // If we're opening docs, hit it and quit.
+    if flags.docs {
+        let docssite = format!("https://www.boltzmann.dev/en/docs/v{}/", version);
+        let subproc = Exec::cmd("open").arg(docssite);
+        subproc.join()?;
+        ::std::process::exit(0);
+    }
+
```

Comment:
I like this a lot! We might wanna flag this as a darwin-only feature, and/or provide equivalents for windows+linux (xdg-open iirc?)